docs/topics/auth/passwords.txt

```diff
@@ -138,14 +183,19 @@ Increasing the work factor
 --------------------------
 
 The PBKDF2 and bcrypt algorithms use a number of iterations or rounds of
-hashing. This deliberately slows down attackers, making attacks against hashed
+hashing.  This deliberately slows down attackers, making attacks against hashed
 passwords harder. However, as computing power increases, the number of
-iterations needs to be increased. We've chosen a reasonable default (and will
-increase it with each release of Django), but you may wish to tune it up or
-down, depending on your security needs and available processing power. To do so,
-you'll subclass the appropriate algorithm and override the ``iterations``
-parameters. For example, to increase the number of iterations used by the
-default PBKDF2 algorithm:
+iterations needs to be increased.  Argon2 has two additional knobs that can
```

Comment:
I'm going to suggest some edits to move argon2 discussion to a separate section with a few todos that I hope can complete: https://dpaste.de/aaF5  You can apply the patch with `patch -p1 -i raw.diff`. Thanks, this is coming along!